msticpy/datamodel/entities/soc/incident.py

```diff
@@ -0,0 +1,169 @@
+# -------------------------------------------------------------------------
```

Comment:
should this be a subfolder of entities or have a separate soc folder under datamodel?
We could put stuff like Alert, Incident, and whatever else (bookmark, observations,...) in there